### DISCUSSION TITLE
A new Version with fancy UI based on fooocus

### DISCUSSION THREAD

**Discussion by coder-zkl1988** [lllyasviel/Fooocus#1952] [Show and tell]

**Comment by mashb1t** [comment]
@zongkelong is there a screenshot missing?

**Reply by docppp** [reply]
you can see in the history of this post that this was some shady link.

**Reply by mashb1t** [reply]
fun fact: the edit history button is not visible on mobile devices... Thanks for the hint, checked on Desktop!
